rpcs3/Crypto/unpkg.cpp

```diff
@@ -45,7 +45,12 @@ bool CheckHeader(const fs::file& pkg_f, PKGHeader* m_header)
 	}
 	}
 
-	if (m_header->header_size != PKG_HEADER_SIZE)
+	if (m_header->header_size == PKG_HEADER_PS1_SIZE)
```

Comment:
I have changed the error message to reflect, that we currently don't support unpacking PS1 .pkg files. (But will hopefully in the future)